scripts/linux/pkg/deb/assets/build.sh

```diff
@@ -64,7 +64,7 @@ else
 fi
 
 
-export DEB_DEPS="${DOTNET_RUNTIME_DEPS}, libc6-dev (>= 2.27)"
+export DEB_DEPS="${DOTNET_RUNTIME_DEPS}, libc6-dev (>= 2.27), adduser (>=3.0)"
```

Comment:
what's so special about adduser <3.0?
Maybe instead we could check which version we have and use it based on the version found on the system?

Reply:
we discussed this

Reply:
`dotnet/runtime-deps:9.0-noble` images does not have adduser package installed by default. Version `3.x` is available on every ubuntu we currently support (focal, jammy, noble).